nixos/modules/services/web-apps/archtika.nix

```diff
@@ -0,0 +1,312 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
```

Comment:
```suggestion
```
Nested `with lib;` is an anti-pattern. Consider using `inherit` in a `let...in` expression, or calling `lib` explicitly.